src/Telegram.Bot/Telegram.Bot.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;netcoreapp3.1</TargetFrameworks>
```

Comment:
.NET Core 3.1 [EOL](https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core) is December 13, 2022 - two weeks from now. Though I don't think we use any specific .NET 6 features I'm not sure we should encourage users to stay on EOL framework. IMO we can safely move to `netstandard2.0;net6.0` target.